server/src/main/scala/org/http4s/server/syntax/package.scala

```diff
@@ -6,7 +6,7 @@ import cats.data.Kleisli
 import cats.implicits._
 
 package object syntax {
-  @deprecated("Import `cats.syntax.semigroup._` and use `service1 |+| service2` instead", "0.16")
+  @deprecated("Import `cats.implicits._` and `org.http4s.implicits._` and use `service1 <+> service2` instead. Ensure -Ypartial-unification is enabled.", "0.16")
```

Comment:
`org.http4s.implicits._` used to be required, but is not as of 0.18.0-M5.  (The semigroupk for kleisli that we got from it is in cats-1.0.0-RC1).